README.md

```diff
@@ -33,3 +99,12 @@ If this project needs something other than the vanilla commands like `forge test
 ## Other stuff
```

Comment:
What's the point of this "Other Stuff" section?

Reply:
Heh, originally the README was more bare so it felt more like a template readme with some defaults you can keep. This included:
- The (still present) "If this project needs something other than the vanilla commands like `forge test`, or needs environment variables or RPC URLs, mention that here." text.
- The "other stuff, do whatever else you need" section

I'm thinking we remove both and make the README purely informational about the template?

Reply:
Updated in 09cf570